python/desc/imsim/skyModel.py

```diff
@@ -62,7 +144,7 @@ def __init__(self, obs_metadata, seed=None, addNoise=True, addBackground=True):
         else:
             self.randomNumbers = galsim.UniformDeviate(seed)
 
-    def addNoiseAndBackground(self, image, bandpass=None, m5=None,
+    def addNoiseAndBackground(self, image, chipName, bandpass=None, m5=None,
```

Comment:
This method is called by `GalSimInterpreter.drawImage` in the `sims_GalSimInterface` package, so we would need to modify that function call in that package to use this code.  We need to discuss with @danielsf  about the implications of this interface change since it would force the `NoiseAndBackgroundBase` class to change.

Reply:
I think this is a great improvement.  If you could just pass `chipName` as a kwarg that defaults to `None`, rather than as a positional arg, we can update `sims_GalSimInterface` so that it expects the kwarg, but places a homogeneous sky background if `chipName=None`.